modules/installation-restricted-network-samples.adoc

```diff
@@ -30,7 +30,7 @@ not addressed in this procedure.
 in the restricted network environment into one of the defined mirrors:
 +
 ----
-$ oc image mirror myregistry.com/myimage:latest myregistry.com/myimage:stable
+$ oc image mirror registry.redhat.io/<myimage>:latest registry.redhat.io/<myimage>:stable
```

Comment:
change `latest` to something that describes the specific tag one is interested in.  It could be `latest`, but it could also be something like `2.5-52` for say the `ruby-25-rhel7` image.

Reply:
maybe have <mytag>, and then elaborate with an example or two in a note what `myimage` and `mytag` could be

Reply:
sorry @bmcelvee realized a couple of more things

- the second param for `oc image mirror` should start with the mirror registry name, not `registry.redhat.io` ... we are employing the `Copy image to another registry` scenario from the `oc help image mirror` doc, not the `Copy image to another tag` that your existing sample seems to have copied
- the `<myimage>:<mytag>` needs to be the same with both parameters for samples import to work ... so basically the registries noted in the 2 params will be different, but everything else will be the same
- some suggestions on how to get the list of images would be useful as well.... I've come up with some prototype `oc` invocations like `oc get is -n openshift -o yaml | grep registry.redhat.io | grep name | | awk '{ print $2 }'` but it still needs a little work ... please devise a placeholder for this while I tinker

Reply:
That's fair. Perhaps it would be better to just use the example @wking provided, "oc image mirror registry.redhat.io/openshift4/ose-jenkins-agent-maven:latest myregistry.com/ose-jenkins-agent-maven:latest "?

Reply:
I included you @bmcelvee on an email I sent to @xiuwang 

I'd like to include some specific examples from her testing

And per our scrum discussion ... most likely punting on the quick command to get the list of possible images to mirror ... that said, I still want to have some follow up discussions with @bparees and @adambkaplan on possibly providing the list in some fashion, most likely separate from this PR ... maybe I manually build the list from a 4.2 cluster and we paste it as an example reference or something

Reply:
When I used payload images to mirror jenkins images to make consistent in the openshift project.
`oc image mirror quay.io/openshift-release-dev/$JENKINSID ${MIRROR_ADDR}/openshift-release-dev/ocp-v4.0-art-dev`
 and for other rhscl  images,
`oc image mirror registry.redhat.io/rhscl/ruby-25-rhel7:latest ${MIRROR_ADDR}/rhscl/ruby-25-rhel7:latest`

Reply:
Thanks for the update @xiuwang 

@bmcelvee - the `MIRROR_ADDR` undoubtedly comes from https://docs.openshift.com/container-platform/4.2/installing/installing_restricted_networks/installing-restricted-networks-preparations.html#installation-creating-mirror-registry_installing-restricted-networks-preparations ... we should cross reference here.

@xiuwang can you clarify how obtained the value for `$JENKINSID` ... it is not obvious to me based on what I see at https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags

Reply:
@gabemontero @bmcelvee I retrieve the jenkins image with SHA id from payload,

  ```listjenkins=`oc adm release info --pullspecs registry.svc.ci.openshift.org/ocp/release:$VERSION | grep  jenkins| awk -F"/" '{print $3}'`
  for JENKINSID in ${listjenkins[@]}
  do  
    oc image mirror quay.io/openshift-release-dev/$JENKINSID ${MIRROR_ADDR}/openshift-release-dev/ocp-v4.0-art-dev
  done
```

Reply:
so yeah as @bparees mentioned below since the jenkins image is in the payload, we should get that one for free when the install payload is mirrored, and you set the `samplesRegistry` setting in the samples operator config to the mirrored registry location.

let's go with the images for the `ruby` imagestream instead ... you can get at either the tags or the sha refs using the script commands I noted in my other comment